packages/plugin-evm/src/actions/swap.ts

```diff
@@ -0,0 +1,106 @@
+import { ChainId, createConfig, executeRoute, getRoutes, ExtendedChain, Chain } from '@lifi/sdk'
+import type { WalletProvider } from '../providers/wallet'
+import type { Transaction, SwapParams, SupportedChain } from '../types'
+import { CHAIN_CONFIGS } from '../providers/wallet'
+
+export const swapTemplate = `Given the recent messages and wallet information below:
```

Comment:
Can we move this template into a templates file and add all templates there for readability? Thank you!